hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OzoneManagerStateMachine.java

```diff
@@ -314,7 +313,11 @@ public TransactionContext preAppendTransaction(TransactionContext trx)
   @Override
   public CompletableFuture<Message> applyTransaction(TransactionContext trx) {
     try {
-      OMRequest request = OMRatisHelper.convertByteStringToOMRequest(
+      // For the Leader, the OMRequest is set in trx in startTransaction.
+      // For Followers, the OMRequest hast to be converted from the log entry.
+      final Object context = trx.getStateMachineContext();
+      final OMRequest request = context != null ? (OMRequest) context
```

Comment:
So, the followers need to deserialize. That's OK if `applyTransaction` on the followers does not slow the commit phase down. Is that the case? 
I think `applyTransaction` is done after the majority commit, just wanna double check.

Reply:
>  ... `applyTransaction` is done after the majority commit, ...

That is correct.